packages/ui/packages/Toggle/src/styles.js

```diff
@@ -1,16 +1,17 @@
-/* eslint-disable prettier/prettier */
 // eslint-disable-next-line import/no-named-default
 import { default as Styled } from 'styled-components';
 const StyledToggleWrapper = Styled.button`
   width: ${props => `calc(${props.theme.defaultRem} * ${props.size} * 2.2)`};
-  height: 100%;
   border-radius: ${props =>
     `calc(${props.theme.defaultRem} * ${props.size + 1})`};
   display: flex;
   padding: 0;
   position: relative;
   outline: none;
   cursor: pointer;
+  ${props => props.disabled ? `
+   filter: grayscale(1);
+  ` : ''}
```

Comment:
this kind of conditions make two classes with all different properties, and cuz re-calculate all properties again. it is better to add a 'disabled` class when want to show disabled thing